### PR TITLE
refactor!: move strprintf(), strscan() and alike from etl to synfig namespace

### PR DESCRIPTION
**Pull request by rodolforg**
BREAKING CHANGE: ETL API changed:
- `vstrprintf()` moved from namespace `etl` to `synfig`
- `strprintf()` moved from namespace `etl` to `synfig`
- `vstrscanf()` moved from namespace `etl` to `synfig`
- `strscanf()` moved from namespace `etl` to `synfig`
- `stratof()` moved from namespace `etl` to `synfig`
- `stratoi()` moved from namespace `etl` to `synfig`

### CODE REVIEW

**Comment by rodolforg** [comment]
Any opinions?

**Comment by ice0** [comment]
> Any opinions?

Looks good to me.

File: synfig-core/src/synfig/string.h [hunk 1]
```diff
@@ -31,6 +31,7 @@
 /* === H E A D E R S ======================================================= */
 
 #include <string>
+#include <synfig/string_helper.h>
```
Comment (rodolforg) [inline synfig-core/src/synfig/string.h:34]: Should I merge those two files? `synfig/string.h` and `synfig/string_helper.h`

Comment (ice0) [reply synfig-core/src/synfig/string.h:34]: I suggest to leave it as is for now.

**Comment by rodolforg** [comment]
Ready to go :)

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**